garak/probes/av_spam_scanning.py

```diff
@@ -6,7 +6,7 @@
 """
 
 from garak import _config
-from garak.probes.base import Probe, Tier
+import garak.probes
```

Comment:
Disprefer import change that requires reference to `Tier` as `garak.probes.Tier` at each usage.

```suggestion
from garak.probes import Probe, Tier
```

Importing from the `package` does seem like an improvement, the trade off for each usage though feels like it would be better off as specific object imports. I don't think this changes much from a plugin author load perspective but not a blocking issue.

Reply:
Disprefer explicit import of Tier, it overstates its importance. Open to other patterns that satisfy this constraint.

Reply:
The form of an import line should not be considered to define the importance of the abstraction it imports. Language syntax does not impart or define value of the class, however it does impact readability and ease of use.

Reply:
That's fair. The ease of use is my originating concern too.